backend/src/models/db/index.js

```diff
@@ -22,7 +22,7 @@ class DB {
     }
 
     deleteDB () {
-        return 0
+        return Promise.resolve(false)
```

Comment:
what's this doing?

Reply:
Normalising the behaviour to reflect what the db does. Which is just return a promise. At the moment the data is nonsense, but might map the args back in future.